new/LIT7226AynatBarya.xml

```diff
@@ -0,0 +1,67 @@
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7226AynatBarya" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Prayer against the evil eye, Bāryā, Legewon, Šotalāy and for pregnant women</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Magic"/>
+                    <term key="Prayers"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2025-01-07">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="dc:relation" active="LIT7226AynatBarya" passive="LIT1763Legend"><desc>The end seems to be similar to 
+                        <ref type="work" corresp="LIT1763Legend"/>.</desc></relation>
+                </listRelation>
+            </div>
+            <div type="edition">
+                <ab>በስመ፡ <gap reason="ellipsis"/> ጸሎት፡ በእንተ፡ ዓይነት፡ <term key="demon">ባርያ፡</term> ወ<term key="demon">ሌጌዎን፡</term>
+                    ወ<term key="demon">ሾተላይ፡</term> በእንተ፡ አሰስሎ፡ <term key="Disease">ደዌ፡ ዘዓቢይ፡ ወዘንዑስ፡</term> ወይበቍዕ፡ አዲ፡ ለብእሲ፡ 
+                    ወለብእሲት፡ ወሚመ፡ ለአእሩግ፡ ወለደቂቅ፡ ወይበእቍዕ፡ ለሕሙማን፡ ወለሕያዋን፡ ወአዲ፡ ለብእሲት፡ ጽንስት፡ ታጽሕፍ፡ ወትሰቅሎ፡ ውስተ፡ ቤታ፡ ወውስተ፡ ውሉዳ፡ ወክሳዳ፡ 
+                    ከመ፡ ይልሕቁ፡ ጽንስት፡ ታጽሕፍ፡ ወትስቅሎ፡ ውስተ፡ ቤታ፡ ወውስተ፡ ውሉዳ፡ ወክሳዳ፡ ከመ፡ ይልሐቁ፡ ደቂቃ፡ ዘልፈ፡ ጸሎቱ፡ ወበረከቱ፡ ለቅዱስ፡ 
```

Comment:
I think some words have been doubled.

If I see correctly, this line should be simply: ከመ፡ ይልሐቁ፡ ደቂቃ፡ ዘልፈ፡ ጸሎቱ፡ ወበረከቱ፡ ለቅዱስ፡